apps/snitch_core/priv/repo/migrations/20180622101026_add_prototype.exs

```diff
@@ -0,0 +1,23 @@
+defmodule Snitch.Repo.Migrations.AddPrototype do
+  use Ecto.Migration
+
+  def change do
+    create table("snitch_option_types") do
+      add :name, :string
+      add :display_name, :string
+      add :position, :integer
+      timestamps()
+    end
+
+    create table("snitch_template_option_values") do
+      add :value, :string
+      add :display_name, :string
+      add :position, :integer
+      add :option_type_id, references("snitch_option_types", on_delete: :delete_all), null: false
+      timestamps()
+    end
+
+    create index("snitch_option_types", [:name])
+    create index("snitch_template_option_values", [:value, :position, :option_type_id])
```

Comment:
Why create indexes now? Have you done any benchmarking and found that these are actually avenues for optimization?
BTW, why is there no unique index on OptionType `:name`?